src/test/resources/application-default.yml

```diff
@@ -0,0 +1,23 @@
+spring:
+  jpa:
+    database: H2
+    properties:
+      dialect: org.hibernate.dialect.H2Dialect
+
+logging:
+  level:
+    org.springframework.web.method.HandlerMethod: TRACE
+    org.springframework.test.web.servlet.TestDispatcherServlet: TRACE
+    liquibase: ERROR #  too verbose to dump liquibase operations at the begining of each test
+    com.zaxxer.hikari.HikariConfig: INFO # too verbose to dump the full config at the begining of each test
+    org.springframework.orm.jpa: DEBUG
+    org.hibernate.SQL: DEBUG
+    org.hibernate.orm.jdbc.bind: TRACE
```

Comment:
maybe we should do this for other repo as well, and change all of them locally when needed

Reply:
some stuff is specific to the way I tested here (redoing the whole liquibase for each test). Some stuff is more general yes